README.md

```diff
@@ -1,25 +1,97 @@
 # prototype-content-consumer
 
-Installation
-=============
 
-$ npm install
+### Background
 
-$ openssl req -nodes -x509 -newkey rsa:2048 -keyout credentials/key.pem -out credentials/cert.pem -days 365
+The project will allow users to capture data on pages and store it to catalogue and organize for later use.
 
-Run
-=============
+### Getting Started
 
-$ sudo npm start
+Follow the steps below to setup a development environment. This project
+expects you will be using a flavor of Unix that supports a standard node.js
+environment. The project has been tested in an OS X 10.11.x environment.
 
-# extension
+_*** the project has not been tested in a windows environment_
 
-Installation
-=============
+1. Clone repository
 
-$ npm install
+    `git clone git@github.com:chadwpry/prototype-content-consumer.git`
 
-Run
-=============
+2. Set NODE_ENV variable to development. It can be included in a .bash_profile or .bashrc configuration.
 
-$ npm run build
+    `$ export NODE_ENV=development`
+
+### API
```

Comment:
Let's keep the API and Extension instructions separate. Move the details for the Extension into a README inside the extension folder.